client/src/components/auth/Login.jsx

```diff
@@ -45,18 +45,19 @@ const Login = () => {
                 setError(error.response.data.message || 'An error occured. Please try again.');
                 console.error('Error response:', error.response.data)
             }
-            setError('An error occurred. Please try again.');
+            setError('Invalid username or password');
             console.error('Error:', error);
         }
     };
```

Comment:
Line 48 is a useState created for error (error, setError) to display a message to the user below the login form, while line 49 is a console logged error. 

In the component, the error is displayed below the form
![Screenshot 2024-11-05 at 3 30 17 PM](https://github.com/user-attachments/assets/66b8b28a-14fc-4c88-900f-5b2221f6950c)


And in the UI it can be shown here.
![Screenshot 2024-11-05 at 3 27 02 PM](https://github.com/user-attachments/assets/a1216267-07a1-485c-9cce-0ce6de8801e5)